packages/design-system/src/components/N8nCallout/Callout.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+	<div :class="classes" role="alert">
+
+		<div :class="$style['message-section']">
+			<div :class="$style.icon">
+				<n8n-icon
+					v-bind="$attrs"
+					:icon="getIcon"
+					:size="theme === 'secondary' ? 'medium' : 'large'"
+				/>
+			</div>
+			<n8n-text
+				size="small"
+				v-bind="$attrs"
+				:bold="theme === 'secondary'"
+			>
+				{{ message }}
+			</n8n-text>
+			<span
+				v-if="actionText"
+				:class="$style['action-text']"
+				@click="$emit('action-text-click')"
+			>
+				{{ actionText }}
+			</span>
```

Comment:
Just like below, should have this inside a default slot for more flexibility. That way, we:
- can have multi-line text
- can have text emphasis (bold, italic)
- can have multiple actions
- don't need to emit custom events